src/Entity/SnapshotManager.php

```diff
@@ -168,4 +168,14 @@ public function createSnapshotPageProxy(TransformerInterface $transformer, Snaps
         return $this->snapshotPageProxyFactory
             ->create($this, $transformer, $snapshot);
     }
+
+    function beginTransaction(): void
+    {
+        $this->getEntityManager()->beginTransaction();
+    }
+
+    function commit(): void
```

Comment:
We should add `rollBack` to the interface too.

(But first the pr should be done on sonata-doctrine-extensions instead)